lighthouse/blueprints/cherrypicked_plates.py

```diff
@@ -169,20 +169,20 @@ def fail_plate_from_barcode() -> FlaskResponse:
 
             return internal_server_error(errors)
 
-        routing_key = get_routing_key(PLATE_EVENT_DESTINATION_FAILED)
+        routing_key = get_routing_key(PE_BECKMAN_DESTINATION_FAILED)
 
         logger.info("Attempting to publish the destination failed event message")
         broker = Broker()
-        broker.connect()
+        broker._connect()
```

Comment:
This is not a private method, why the underscore in the naming?

Reply:
I need to add comments for these changes.

Reply:
Comments added.